newsfragments/29395.significant.rst

```diff
@@ -0,0 +1,7 @@
+The date-time fields passed as API parameters or Params should be RFC3999-compliant.
+
+In case of API calls, it was possible that "+" passed as part of the date-time fields were not URL-encoded, and
+such date-time fields could pass validation. Such date-time parameters should now be URL-encoded (as ``%2B``).
```

Comment:
OMG! I've broke my head when I tried to understand why on Earth `tests/api_connexion/endpoints/test_dag_run_endpoint.py::TestGetDagRunsPaginationFilters::test_date_filters_gte_and_lte` failed what the component changed `2020-06-18T18:00:00+00:00` to `2020-06-18T18:00:00 00:00`